app/src/main/java/com/practicum/playlistmaker/creator/Creator.kt

```diff
@@ -17,16 +40,46 @@ object Creator {
     private fun provideMediaPlayerRepository(): MediaPlayerRepositoryImpl {
```

Comment:
```suggestion
    private fun provideMediaPlayerRepository(): MediaPlayerRepository {
```
Нужно везде с интерфейсами работать и максимально скрывать реализацию

Reply:
Сделал